storm-client/test/jvm/org/apache/storm/security/auth/authorizer/DRPCSimpleACLAuthorizerTest.java

```diff
@@ -68,101 +68,101 @@ private static ReqContext makeMockContext(String user) {
     @Test
     public void test_partial_authorization() {
 
-        Assert.assertFalse("Deny execute to unauthroized user",
-                           isPermitted(strictHandler, ReqContext.context(), "execute", partialFunction));
+        assertFalse(isPermitted(strictHandler, ReqContext.context(), "execute", partialFunction),
+            "Deny execute to unauthorized user");
 
-        Assert.assertTrue("Allow execute to authorized kerb user for correct function",
-                          isPermitted(strictHandler, aliceKerbContext, "execute", partialFunction));
+        assertTrue(isPermitted(strictHandler, aliceKerbContext, "execute", partialFunction),
+            "Allow execute to authorized kerb user for correct function");
 
-        Assert.assertFalse("Deny fetchRequest to unauthorized user for correct function",
-                           isPermitted(strictHandler, aliceKerbContext, "fetchRequest", partialFunction));
+        assertFalse(isPermitted(strictHandler, aliceKerbContext, "fetchRequest", partialFunction),
+            "Deny fetchRequest to unauthorized user for correct function");
```

Comment:
negate error message on this and others in this file as appropriate

Reply:
done